src/main/groovy/wooga/gradle/version/VersionPlugin.groovy

```diff
@@ -83,6 +83,7 @@ class VersionPlugin implements Plugin<Project> {
             def rawValue = VersionPluginConventions.versionCodeOffset.getValue(project)
             Integer.parseInt(rawValue.toString())
         }))
-        extension
+        extension.prefix.convention("v")
```

Comment:
We don't want to implement our usual fallback to env/property flow? I see no harm of allowing the prefix to be set via environment.

Reply:
This is addressed in #60. I tried to make this PR fit the same code style as the current one in atlas version to minimize the changes. Then later I change the style for everyone.